src/NewTools-Debugger/StDebuggerContextInteractionModel.class.st

```diff
@@ -51,8 +51,8 @@ StDebuggerContextInteractionModel >> doItReceiver [
 
 { #category : #testing }
 StDebuggerContextInteractionModel >> hasBindingOf: aString [
-
-	^ (context lookupVar: aString) notNil
+	"The debugger does not define new bindings"
+	^nil
```

Comment:
it seems it is an error. It should return false.

Reply:
ups. yes